reframe/utility/cpuinfo.py

```diff
@@ -281,7 +281,9 @@ def _sysctl_topo():
 
 def cpuinfo():
     ret = {
-        'arch': archspec.cpu.host().name
+        'arch': archspec.cpu.host().name,
+        'vendor': archspec.cpu.host().vendor,
+        'model': archspec.cpu.detect.raw_info_dictionary()['model name']
```

Comment:
Maybe retrive the model name with `.get` and add a default value of Unknown for example here? What do you think @vkarak

Reply:
Yes, because otherwise it will crash if for some reason the `model name` key is not there.

Reply:
Makes sense, will do!

Reply:
addressed via [0c07e8c](https://github.com/reframe-hpc/reframe/pull/3107/commits/0c07e8c31b7447a2bfe190466b25bbb22f0a6c6d)

Reply:
Good call @teojgo 😄 --
https://github.com/archspec/archspec/pull/149 looks like it'll probably break `archspec.cpu.detect.raw_info_dictionary()`. 

@vkarak / @teojgo, how can I best help update this line when RFM bumps the pinned version of archspec?

Reply:
The unit tests run the cpu auto-detection code, so I hope that this will be caught from the CI when the version is bumped. I could tag you in that case, if the fix is not obvious.